WeScan/Scan/CaptureSessionManager.swift

```diff
@@ -144,10 +147,60 @@ final class CaptureSessionManager: NSObject, AVCaptureVideoDataOutputSampleBuffe
             return
         }
         
-        let videoOutputImage = CIImage(cvPixelBuffer: pixelBuffer)
-        let imageSize = videoOutputImage.extent.size
+        var motion: CMMotionManager!
+        motion = CMMotionManager()
+        motion.accelerometerUpdateInterval = 0.2
+        motion.startAccelerometerUpdates(to: OperationQueue()) { p, _ in
```

Comment:
I'd be in favor of using descriptive variable names here instead of `a` and `p`.

Reply:
Done (bb2db9d)